src/test/java/apoc/load/LoadGoogleCloudStorageTest.java

```diff
@@ -0,0 +1,70 @@
+package apoc.load;
+
+import apoc.util.TestUtil;
+import org.junit.*;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.test.TestGraphDatabaseFactory;
+
+import static apoc.load.LoadCsvTest.assertRow;
+import static apoc.util.MapUtil.map;
+import static apoc.util.TestUtil.testResult;
+import static org.junit.Assert.assertFalse;
+
+public class LoadGoogleCloudStorageTest {
+
+    private GraphDatabaseService db;
+
+    private static final String BUCKET_NAME = System.getenv("APOC_GC_BUCKET_NAME");
```

Comment:
where does this come from?

Reply:
Is this something we can set up dynamically?

Reply:
Unfortunately (as the result of my investigation) there is no Docker container or java library that allows testing the connection to a Google Storage Bucket. The only way to do that at this moment is to connect to a real bucket. So I used my private bucket to test it. I thought that we can use an env variable so into Travis we can easily inject a bucket.

Reply:
ok